libraries/rush-lib/src/logic/buildCache/ProjectBuildCache.ts

```diff
@@ -13,33 +13,37 @@ import type { ICloudBuildCacheProvider } from './ICloudBuildCacheProvider';
 import type { FileSystemBuildCacheProvider } from './FileSystemBuildCacheProvider';
 import { TarExecutable } from '../../utilities/TarExecutable';
 import { EnvironmentVariableNames } from '../../api/EnvironmentConfiguration';
+import type { Operation } from '../operations/Operation';
 
-export interface IProjectBuildCacheOptions {
+export interface IOperationBuildCacheOptions {
   /**
    * The repo-wide configuration for the build cache.
    */
   buildCacheConfiguration: BuildCacheConfiguration;
-  /**
-   * The project to be cached.
-   */
-  project: RushConfigurationProject;
   /**
    * Value from rush-project.json
    */
   projectOutputFolderNames: ReadonlyArray<string>;
```

Comment:
This is just the value `[executionRecord.metadataFolderName, ...(executionRecord.operation.settings?.outputFolderNames ?? [])]`